src/components/Invoices/index.js

```diff
@@ -0,0 +1,99 @@
+import InvoiceCards from "./InvoiceCards";
+import NewInvoice from "./NewInvoice";
+import ViewInvoice from "./ViewInvoice";
+import {useState, useEffect} from "react";
+
+
+const Invoices = () => {
+
+    const [invoices, setInvoices] = useState([]);
+
+    // --- fetch invoices and returns json promise ---
+    const extractResponseData = (response) => {
+        return response.json();
+    }
+
+    const fetchInvoices = async () => {
+        const response = await fetch('http://localhost:8080/invoices');
+        if (!response.ok) {
+            throw new Error();
+        }
+        return await extractResponseData(response);
+    }
+
+    // --- on mount logs the data retrieved ---
+    // --- Replace console.log(invoiceData) with setState inside the .then() to display data ---
```

Comment:
Looks like this has been done, so the comment is redundant now. Remove it.

Reply:
done